spec/cucumber/core/test/filters/locations_filter_spec.rb

```diff
@@ -102,57 +102,131 @@ module Core
                   | 1 | 2 |
                   | 3 | 4 |
 
+              Rule: A rule with a background
+                Background: A background rule
+                  Given background
+
+                Scenario: with a rule and background
+                  Given rule a
+
+                Scenario: another with a rule and background
+                  Given rule a
+
+              Rule: A rule without a background
+                Scenario: with a rule and no background
+                  Given rule a
```

Comment:
can we rename this portion of the test just for documentation purposes, to say something like `Given a rule without a background`